codespell_lib/data/dictionary.txt

```diff
@@ -1509,7 +1513,7 @@ diferently->differently
 diferrent->different
 differance->difference
 differentiatiations->differentiations
-differents->different
+differents->difference
```

Comment:
it should probably have both, with the old correction as the first choice

Reply:
done